docs/osf-trivia-list/dasharo.md

```diff
@@ -235,6 +235,81 @@ Dasharo Tools Suite
 * We provide ready to use binaries with GPG based signing scheme that improve
   verification where firmware coming from.
 
+## What are the differences between the official coreboot repository and the Dasharo repository?
+
+* Dasharo focus on specific platforms listed in  [supported
+  hardware](https://docs.dasharo.com/unified/novacustom/overview/) section of Dasharo
+  Universe documentation.
+
+Dasharo repository contains release tags which are associated with Dasharo
+Certification Program providing certain quality criteria including test
+results. We always trying to minimize delta, but sometimes it can be up to 5k
+SLOC (or more I guess e.g. Talos II coreboot support).
+
+## What is Dasharo Certification Program?
+
+Dasharo Certification Program (DCP) is a rigorous certification program that
+sets a high-quality bar for open-source firmware. It aims to provide stable,
+secure, and reliable firmware that meets the Dasharo values. The program
+encourages developers to create their forks of Dasharo or contribute back to
+the Dasharo project or coreboot upstream. By doing so, they can leverage the
+power of open-source development to create custom firmware tailored to their
+specific needs. The certification process ensures that the firmware binary
+meets the strict criteria by testing  3mdeb in the certification lab.
+
+The Dasharo-certified firmware provides long-term maintenance over ten years
+after the CPU microarchitecture release, which means that OEM, ODM, hardware
+vendors, and other companies can rely on the firmware for a long time without
+worrying about end-of-life issues. Moreover, DCP-approved firmware vendors must
+provide professional support channels to ensure that other business entities
+can rely on those channels for long-term support regarding firmware and
+software.
+
+The validation process for DCP firmware is transparent. Test results and bug
+reports are always publicly available, allowing the community to continually
+identify issues and improve the firmware. However, in case of a security
+embargo, the results can be kept under a strict but well-defined policy,
+ensuring the security of the firmware.
+
+DCP-approved firmware respects privacy by allowing the deactivation of
```

Comment:
Is it mandatory to be DCP-approved? Not all released firmware currently provide such functionality, even `permitted by the current state of technology`

Reply:
@macpijan IMO that is bad and should be treated as bug. We should provide that choice otherwise platform is not fully DCP-approved and we should clearly communicate to users that given feature is missing.

In long run vision is to have highest level of DCP at bleeding edge of Qubes OS hardware certification requirements (e.g. DMA protection, USB and Network stack disabled etc.).

This is WIP idea, but if you think we should approach the problem differently please let me know.

Reply:
@macpijan WDYT?

Reply:
@macpijan please let me know if this works for you: https://github.com/Dasharo/docs/pull/507/commits/9f53835fe3d5e1dc2f18fd65021cb9b99bdabdf7